lib/HTML/Renderers/TocNodeRenderer.php

```diff
@@ -49,11 +54,43 @@ public function render(): string
             $url = $this->environment->relativeUrl($reference->getUrl());
 
             $titleLinkUtility->buildLevel($url, $reference->getTitles(), 1, $tocItems, $file);
+            if ($maxDepth !== 0 && $maxDepth <= 1) {
```

Comment:
```suggestion
            if ($maxDepth === 1) {
```

(unless I'm missing something)